overops-functions/src/main/resources/META-INF/UDF_MANIFEST.MF

```diff
@@ -43,11 +43,11 @@
 	      <param_type>TEXT</param_type>
 	      <class_file>com.takipi.udf.volume.RegressionFunction</class_file>
 	      <default_params>
-	      	# the active time window (min) to inspect
-	      	activeTimespan = 1440
+	      	# the active time window to inspect
+	      	activeTimespan = 1d
```

Comment:
1. Why is this not in another PR that actually refers to this funcion?
2. Are you sure that the code can handle this?